stl/inc/bitset

```diff
@@ -400,7 +401,18 @@ public:
     }
 
     _NODISCARD bool all() const noexcept {
-        return count() == size();
+        if _CONSTEXPR_IF (_Bits == 0) {
+            return true;
+        }

```

Comment:
Do we need a special optimized implementation for `bitset<0>`? I've never seen that in user code, only in libcxx's test suite. If the general implementation works, I think that a special case is unnecessary here. (Note that `any()` doesn't have such a special case.)

Reply:
I don't think general implementation works as it is currently written. 
I think it would have failed by calculating bits in remainder.